Lib/typing.py

```diff
@@ -1862,8 +1862,6 @@ def meth(self) -> T:
                 ...
     """
     __slots__ = ()
-    _is_protocol = True
```

Comment:
Why are these removed? I bet they're pretty commonly used for introspection.

Reply:
Because we didn't need them, and it seemed cleaner without them. But I agree that we shouldn't unnecessarily break users. Added them back in 36b2ac9.